src/views/TradingCompetition/components/TeamRanks/TopTradersCard/GridItem.tsx

```diff
@@ -28,7 +28,7 @@ const Item = styled(Flex)`
   }
 `
 
-const GridItem: React.FC<{ traderData?: LeaderboardDataItem; index }> = ({ traderData, index }) => {
+const GridItem: React.FC<{ traderData?: LeaderboardDataItem; index?: number }> = ({ traderData, index }) => {
```

Comment:
Make this required or give it a default value.

Reply:
Done 👍